kernel/base/src/main/java/com/twosigma/beaker/jvm/object/HelpLink.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
just use a helper function in KernelInfoHandler.java, this class it too lightweight.